src/components/Continuous.tsx

```diff
@@ -1,32 +1,175 @@
-import { Paper } from '@mui/material';
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Autocomplete,
+  TextField,
+} from '@mui/material';
+import { useEffect } from 'react';
+import useDataStore from '~/stores/data';
+import { StandardizedVariable } from '~/utils/types';
 import DescriptionEditor from './DescriptionEditor';
+import MissingValueButton from './MissingValueButton';
 
 interface ContinuousProps {
   columnID: string;
   units: string;
-  onUpdateUnits: (columnId: string, units: string) => void;
+  uniqueValues: string[];
+  missingValues: string[];
+  format?: {
+    termURL?: string;
+    label?: string;
+  };
+  standardizedVariable?: StandardizedVariable | null;
+  onUpdateUnits: (columnID: string, units: string) => void;
+  onToggleMissingValue: (columnID: string, value: string, isMissing: boolean) => void;
+  onUpdateFormat: (columnID: string, format: { termURL: string; label: string } | null) => void;
 }
 
-function Continuous({ columnID, units, onUpdateUnits }: ContinuousProps) {
+const defaultProps = {
+  standardizedVariable: null,
+  format: null,
+};
+
+const formatOptions = [
+  { label: 'euro', value: 'nb:FromEuro' },
+  { label: 'bounded', value: 'nb:FromBounded' },
+  { label: 'float', value: 'nb:FromFloat' },
+  { label: 'iso8601', value: 'nb:FromISO8601' },
+  { label: 'range', value: 'nb:FromRange' },
+];
+
+function Continuous({
+  columnID,
+  units,
+  uniqueValues,
+  missingValues,
+  format,
+  standardizedVariable,
+  onUpdateUnits,
+  onToggleMissingValue,
+  onUpdateFormat,
+}: ContinuousProps) {
+  useEffect(() => {
+    // Reset scroll position when column changes
+    const tableContainer = document.getElementById(`${columnID}-table-container`);
+    if (tableContainer) {
+      tableContainer.scrollTop = 0;
+    }
+  }, [columnID]);
+
+  const { getAssessmentToolConfig } = useDataStore();
+
+  const showUnit = standardizedVariable?.identifier !== getAssessmentToolConfig().identifier;
+  const showFormat =
+    standardizedVariable &&
+    standardizedVariable?.identifier !== getAssessmentToolConfig().identifier;
```

Comment:
is this line not identical to the `showUnit` you just declared?

Reply:
maybe it's late, but I don't see how `showUnit` and `showFormat` can ever have different values. I.e. how one can be `true` if the other is `false`. So please take another look and remove one if you agree (also likely rename in that case).

Reply:
> is this line not identical to the showUnit you just declared?

So this was implemented right before the demo when we deceided to switch the dataType of Assessment tool standardized variable to continuous and that would be the only case where the `showUnit` and `showFormat` would be different.
I've opened an issue to discuss the assessment tool (and multi column measure standardized variables) data type here #150